dist/js/mdDateTimePicker.js

```diff
@@ -1,32 +1,22 @@
 (function (global, factory) {
 	if (typeof define === "function" && define.amd) {
-		define(['exports', 'moment', 'Draggabilly'], factory);
```

Comment:
@netTrek  there were some changes that I did in my master please fix them from here:
https://github.com/puranjayjain/md-date-time-picker/blob/master/src/js/mdDateTimePicker.js#L12
https://github.com/puranjayjain/md-date-time-picker/blob/master/src/js/mdDateTimePicker.js#L13
https://github.com/puranjayjain/md-date-time-picker/blob/master/src/js/mdDateTimePicker.js#L1367
or you could just pull from upstream
http://stackoverflow.com/questions/6102297/how-do-i-clean-up-my-github-fork-so-i-can-make-clean-pull-requests